internal/tree.go

```diff
@@ -12,7 +12,23 @@ import (
 	"github.com/xlab/treeprint"
 )
 
-func renderTreeView(tasks []task) error {
+var (
+	Name           string
+	Version        string
```

Comment:
These variables are provided through the Makefile, and are not used in any of the commands - only in `checkpointctl version`. It would be better to have this in `checkpointctl.go`, since it has nothing to do with the tree view.